### PR TITLE
chore(condo): DOMA-10531 meters billing integration improvements

### PR DESCRIPTION
**Pull request by AleX83Xpert**

### CODE REVIEW

File: apps/condo/domains/meter/constants/constants.js [hunk 1]
```diff
@@ -11,6 +11,7 @@ const MESSENGER_METER_READING_SOURCE_ID = 'e5f3d3d3-abc2-48a8-99e8-33ade60ea205'
 const SOCIAL_NETWORK_METER_READING_SOURCE_ID = 'e07c307f-1c7a-4da6-87b9-1c03a990a19d'
 const MOBILE_APP_STAFF_METER_READING_SOURCE_ID = '43c9fd8e-802f-4e8a-98b9-532596d7f8c1'
 const CRM_METER_READING_SOURCE_ID = 'b09fec04-5bc2-4c58-a364-04b1ea505503'
+const METERS_BILLING_INTEGRATION_KVP24_READING_SOURCE_ID = '3450106c-2613-49e8-8d97-b3adf5edd189'
```
Comment (toplenboren) [inline apps/condo/domains/meter/constants/constants.js:14]: I do not really like the idea of moving some internal kvp24 stuff (it is a b2b app afaik) to open source condo part

File: apps/condo/domains/meter/schema/MeterReading.js [hunk 2]
```diff
@@ -19,7 +19,11 @@ const { Meter } = require('@condo/domains/meter/utils/serverSchema')
 const { connectContactToMeterReading } = require('@condo/domains/meter/utils/serverSchema/resolveHelpers')
 const { addClientInfoToResidentMeterReading } = require('@condo/domains/meter/utils/serverSchema/resolveHelpers')
 const { addOrganizationFieldPlugin } = require('@condo/domains/organization/schema/plugins/addOrganizationFieldPlugin')
-const { RESIDENT } = require('@condo/domains/user/constants/common')
+const { RESIDENT, SERVICE } = require('@condo/domains/user/constants/common')
+
+function canEditBillingStatusFields ({ authentication: { item: user } }) {
```
Comment (toplenboren) [inline apps/condo/domains/meter/schema/MeterReading.js:24]: I thought we put all access related stuff to ./access

File: apps/condo/lang/en/en.json [hunk 3]
```diff
@@ -1764,6 +1764,7 @@
   "meterReadingSource.Messenger.name": "Messengers (tg, whatsapp, viber, ..)",
   "meterReadingSource.SocialNetwork.name": "Social media (vk, fb, ..)",
   "meterReadingSource.MobileAppStaff.name": "Staff mobile app",
+  "meterReadingSource.metersBillingIntegrationKvp24.name": "Integration with kvp24",
```
Comment (toplenboren) [inline apps/condo/lang/en/en.json:1767]: it is best to read it dynamically from connected b2b app

**Review by toplenboren — CHANGES_REQUESTED** [review]

File: apps/condo/domains/meter/constants/constants.js (hunk 1 above)
Comment (AleX83Xpert) [reply apps/condo/domains/meter/constants/constants.js]: You right& Renamed to "external integration"

File: apps/condo/domains/meter/schema/MeterReading.js [hunk 4]
```diff
@@ -19,7 +19,11 @@ const { Meter } = require('@condo/domains/meter/utils/serverSchema')
...
+    return user.isAdmin || user.isSupport || user.type === SERVICE
```
Comment (SavelevMatthew) [inline apps/condo/domains/meter/schema/MeterReading.js:25]: Any service?

Comment (SavelevMatthew) [reply apps/condo/domains/meter/schema/MeterReading.js:25]: Don't you need to specify, what user exactly should do?

File: apps/condo/domains/meter/schema/MeterReading.js (hunk 2 above)
Comment (AleX83Xpert) [reply apps/condo/domains/meter/schema/MeterReading.js]: Moved to the "access" folder

File: apps/condo/lang/en/en.json (hunk 3 above)
Comment (AleX83Xpert) [reply apps/condo/lang/en/en.json]: Just renamed to "External integration".
But what do you mean?

File: apps/condo/domains/meter/schema/MeterReading.js (hunk 4 above)
Comment (AleX83Xpert) [reply apps/condo/domains/meter/schema/MeterReading.js]: Only [who can edit the model](https://github.com/open-condo-software/condo/blob/e9d5784dc9c04716df8294026fe251477ee05a8a/apps/condo/domains/meter/access/MeterReading.js#L62).
Just added the test for it.

File: apps/condo/domains/meter/constants/constants.js [hunk 5]
```diff
@@ -36,6 +37,7 @@ const METER_READING_SOURCE_MESSENGER_TYPE = 'messenger'
 const METER_READING_SOURCE_SOCIAL_NETWORK_TYPE = 'social_network'
 const METER_READING_SOURCE_MOBILE_APP_STAFF_TYPE = 'mobile_app_staff'
 const METER_READING_SOURCE_CRM_TYPE = 'crm'
+const METER_READING_SOURCE_EXTERNAL_INTEGRATION_TYPE = 'external_integration'
```
Comment (toplenboren) [inline apps/condo/domains/meter/constants/constants.js:40]: How will you get what external integration created the reading? `sender` and `updatedBy`?

**Review by toplenboren — APPROVED** [review]

File: apps/condo/domains/meter/constants/constants.js (hunk 5 above)
Comment (AleX83Xpert) [reply apps/condo/domains/meter/constants/constants.js:40]: > `sender` and `updatedBy`

Yes

**Review by dkoviazin — APPROVED** [review]

**Pull request merged**